### PR TITLE
Using torrent_date instead of timestamp for channel metadata update time

### LINKED ISSUE #4326: ValueError: year is out of range
After rebasing to new devel:
```python
ERROR   1551977540.15            tribler_window:67   (root)  Traceback (most recent call last):
  File "/home/vader/src/TRIBLER/tribler_ichorid/TriblerGUI/widgets/tablecontentmodel.py", line 91, in data
    if column in self.column_display_filters else data
  File "/home/vader/src/TRIBLER/tribler_ichorid/TriblerGUI/widgets/tablecontentmodel.py", line 168, in <lambda>
    u'updated': lambda date: pretty_date(date // 1000),
  File "/home/vader/src/TRIBLER/tribler_ichorid/TriblerGUI/utilities.py", line 96, in pretty_date
    diff = now - datetime.fromtimestamp(time)
ValueError: year is out of range
```

### PR DESCRIPTION
**Pull request by devos50**
Also not crashing the `pretty_date` method when an invalid time is passed to the method.

Fixes #4326

### CODE REVIEW

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

File: Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py [hunk 1]
```diff
@@ -450,7 +452,7 @@ def to_simple_dict(self):
                 "subscribed": self.subscribed,
                 "votes": self.votes,
                 "status": self.status,
-                "updated": self.timestamp,
+                "updated": int((self.torrent_date - epoch).total_seconds() * 1000.0),
```
Comment (ichorid) [inline Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py]: You don't even need to `*1000.0`. `torrent_date` is seconds-granularity (as in .torrent files).

**Comment by devos50** [comment]
retest this please

**Pull request merged**